pydicom/tag.py

```diff
@@ -166,16 +167,24 @@ def __ge__(self, other: object) -> bool:
 
     def __gt__(self, other: object) -> bool:
         """Return ``True`` if `self` is greater than `other`."""
+        # Check if comparing with another Tag object; if not, create a temp one
+        if not isinstance(other, BaseTag):
```

Comment:
Not needed

Reply:
I haven't seen how > is implemented as a combination of == and <. You are right.
Sorry for not seeing the details.